.github/workflows/cleanup.yaml

```diff
@@ -0,0 +1,44 @@
+name: Сleanup caches
+
+on:
+  workflow_call:
+    secrets:
+      DOCKER_REGISTRY_USER:
+        required: true
+      DOCKER_REGISTRY_PASSWORD:
+        required: true
+
+jobs:
+  cleanup:
+    name: Сleanup caches
+    runs-on: [self-hosted, common]
+    container:
+      image: europe-docker.pkg.dev/ftcs-production/infrastructure/node-build-tools:cleanup
```

Comment:
IMO it should be separate image name, not image tag because you use completely separate image, not node-build-tools

Reply:
changed to europe-docker.pkg.dev/ftcs-production/infrastructure/github-actions-cleanup:latest